db/people.yml

```diff
@@ -858,6 +858,33 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: scientist
+dpeters:
+  active: true
+  aka: Dom
```

Comment:
this should be a list, not a string, so it would look like
```
aka:
  - Dom
 ```

Reply:
Done.